packages/react/test/evaluation.spec.tsx

```diff
@@ -35,369 +41,540 @@ describe('evaluation', () => {
-          <React.Suspense fallback={<div>{FALLBACK}</div>}>
-            <TestComponent></TestComponent>
-          </React.Suspense>
-        </OpenFeatureProvider>,
-      );
 
-      // initially should be default, because suspendUntilReady={false}
-      expect(screen.queryByText(DEFAULT)).toBeInTheDocument();
+    describe('suspend=false', () => {
+      it('should not suspend until reconciled and then render', async () => {
+        await OpenFeature.setContext(SUSPENSE_OFF, {});
+        OpenFeature.setProvider(SUSPENSE_OFF, suspendingProvider());
+
+        render(
+          // disable suspendUntilReady, we are only testing reconcile suspense.
+          <OpenFeatureProvider domain={SUSPENSE_OFF} suspend={false}>
+            <React.Suspense fallback={<div>{FALLBACK}</div>}>
+              <TestComponent></TestComponent>
+            </React.Suspense>
+          </OpenFeatureProvider>,
+        );
+
+        // assert no suspense
+        expect(screen.queryByText(DEFAULT)).toBeInTheDocument();
+        expect(screen.queryByText(FALLBACK)).toBeNull();
+
+        // expect to see static value after we are ready
+        await waitFor(() => expect(screen.queryByText(STATIC_FLAG_VALUE_A)).toBeInTheDocument(), {
+          timeout: DELAY * 2,
+        });
 
-      // update the context without awaiting
-      act(() => {
-        OpenFeature.setContext(SUSPENSE_ON, { user: TARGETED_USER });
-      });
+        // update the context without awaiting
+        act(() => {
+          OpenFeature.setContext(SUSPENSE_OFF, { user: TARGETED_USER });
+        });
 
-      // expect to see fallback while we are reconciling
-      await waitFor(() => expect(screen.queryByText(FALLBACK)).toBeInTheDocument(), { timeout: DELAY / 2 });
+        // expect to see static value until we reconcile
+        await waitFor(() => expect(screen.queryByText(STATIC_FLAG_VALUE_A)).toBeInTheDocument(), {
+          timeout: DELAY / 2,
+        });
 
-      // make sure we updated after reconciling
-      await waitFor(() => expect(screen.queryByText(TARGETED_FLAG_VALUE)).toBeInTheDocument(), {
-        timeout: DELAY * 2,
+        // make sure we updated after reconciling
+        await waitFor(() => expect(screen.queryByText(TARGETED_FLAG_VALUE)).toBeInTheDocument(), {
+          timeout: DELAY * 2,
+        });
       });
     });
   });
 
-  describe('suspend=false', () => {
-    it('should not suspend until reconciled and then render', async () => {
-      await OpenFeature.setContext(SUSPENSE_OFF, {});
-      OpenFeature.setProvider(SUSPENSE_OFF, suspendingProvider());
-
-      render(
-        // disable suspendUntilReady, we are only testing reconcile suspense.
-        <OpenFeatureProvider domain={SUSPENSE_OFF} suspend={false}>
-          <React.Suspense fallback={<div>{FALLBACK}</div>}>
-            <TestComponent></TestComponent>
-          </React.Suspense>
-        </OpenFeatureProvider>,
-      );
+  describe('context, hooks and options', () => {
```

Comment:
These are the new tests - they are not DOM-based, but make assertions about the associated client calls using jest spies.